test/config/logger.config,js

```diff
@@ -0,0 +1,12 @@
+const winston = require('winston');
+
+const logger = winston.createLogger({
+    level: 'debug',
+    transports: [
+        new winston.transports.Console({ level: 'info' }),
+        new winston.transports.File({ filename: 'combined.log', level: 'debug'})
+    ],
+    format: winston.format.simple()
+});
+
+module.exports = logger;
```

Comment:
What is the point of creating this reporter if you don't use it anywhere?

Reply:
you are right. Logger will be added further in the Module #8 so logger is removed from this task